### DISCUSSION TITLE
Blank page for search results

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#3186] [Q&A] [answered]
If you have a page where the primary user action is to search and the user has not searched yet, should the content below be empty, use an empty state component to indicate that the user has not yet typed in an entry, or use a different component?  See below for screenshot:

<img width="1431" alt="Screenshot 2023-04-21 at 12 17 19 PM" src="https://user-images.githubusercontent.com/9394413/233685614-5634ec16-691d-40d6-9f55-47bcce3274a6.png">

Any advice would be super helpful, thank you!

**Comment by TheSisb** [comment, marked as answer]
Hi @tonge495,

This is a good example of where the [Empty State Pattern](https://paste.twilio.design/patterns/empty-state) can come in handy. That would be my recommendation here.

**Reply by tonge495** [reply]
Thank you!
